.github/workflows/build.yml

```
@@ -0,0 +1,61 @@
+name: Build
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        include:
+          - name: macOS 10.15
+            os: macos-10.15
```

```suggestion
          - name: macOS 11.0
            os: macos-11.0
```
Any particular reason for using 10.15?

---

No, copy & paste from the main mixxx repo. Are you sure this doesn't cause issues when building mixxx?

---

Mixxx builds with macOS 11.0 but CPack requires CMake 3.19 to work on macOS 11: https://gitlab.kitware.com/cmake/cmake/-/issues/21572